libaleth/SendDialog.ui

```diff
@@ -2,6 +2,32 @@
 <ui version="4.0">
  <class>SendDialog</class>
  <widget class="QDialog" name="SendDialog">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>223</width>
```

Comment:
I think it's not a good idea to fix the pixel size of anything. What if the font size is different?